react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecText.java

```diff
@@ -195,13 +208,21 @@ public boolean requestFocus(int direction, Rect previouslyFocusedRect) {
             return false;
         }*/
         setFocusableInTouchMode(true);
+        setFocusable(true);
         boolean focused = super.requestFocus(direction, previouslyFocusedRect);
         showSoftKeyboard();
         return focused;
     }
 
-    private boolean showSoftKeyboard() {
-        return mInputMethodManager.showSoftInput(this, 0);
+    private void showSoftKeyboard() {
```

Comment:
Just curious, is this to ensure `showSoftInput` is called on the UI thread?

Reply:
Nope, is for scheduling the call on a next frame.

Reply:
Ah, thanks for the explanation :+1: